extensions/keycloak-authorization/runtime/src/main/java/io/quarkus/keycloak/pep/runtime/KeycloakPolicyEnforcerConfig.java

```diff
@@ -41,7 +41,7 @@
          * Specifies how policies are enforced.
          */
         @ConfigItem(defaultValue = "ENFORCING")
```

Comment:
Could you make `enforcing` lowercase? For enums, the doc will advertise the values in lowercase with dashes so better have the default value consistent.